server/plugin/command.go

```diff
@@ -456,6 +456,15 @@ func (p *Plugin) handleIssue(_ *plugin.Context, args *model.CommandArgs, paramet
 type CommandHandleFunc func(c *plugin.Context, args *model.CommandArgs, parameters []string, userInfo *GitHubUserInfo) string
 
 func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
+	config := p.getConfiguration()
+
+	if err := config.IsValid(); err != nil {
+		githubPluginURL := *p.API.GetConfig().ServiceSettings.SiteURL + "/admin_console/plugins/plugin_github"
+		text := fmt.Sprintf(" Before using this plugin, you will need to configure it by filling out the settings in the system console [%s](%s). You can learn more about the setup process [%s](%s) .", "here", githubPluginURL, "here", "https://github.com/mattermost/mattermost-plugin-github#step-3-configure-the-plugin-in-mattermost")
```

Comment:
At least for admin it would make sense to also print out the error. For non-admin this might leak information and hence the code needs to check the user role.

Reply:
do we have any existing code which check the user role ?

Reply:
Here's an excerpt from the Jira plugin https://github.com/mattermost/mattermost-plugin-jira/blob/HEAD/server/command.go#L761

Reply:
For a non-admin, we can show the message "Please contact your system administrator to configure the GitHub plugin."

Reply:
yes , implemented check for admin user